tests/SearchTest.java

```diff
@@ -1,29 +1,144 @@
 import static org.junit.Assert.*;
 
+import app.SearchUseCaseFactory;
+import app.SwitchViewUseCaseFactory;
 import data_access.ElasticsearchDataAccessObject;
 import entities.search.SearchChatMessage;
 import entities.search.SearchRequest;
+import interface_adapter.ViewManagerModel;
+import interface_adapter.search.*;
+import interface_adapter.searched.SearchedViewModel;
+import interface_adapter.switch_view.SwitchViewController;
+import java.awt.*;
 import java.time.Instant;
+import javax.swing.*;
 import okhttp3.OkHttpClient;
+import org.junit.Before;
 import org.junit.Test;
+import view.*;
 
 public class SearchTest {
   private final OkHttpClient okHttpClient = new OkHttpClient();
-  private final ElasticsearchDataAccessObject searchDataAccessObject =
-      new ElasticsearchDataAccessObject(okHttpClient);
+
+  ElasticsearchDataAccessObject searchDataAccessObject;
+
+  SearchViewModel searchViewModel;
+
+  SearchedViewModel searchedViewModel;
+  ViewManagerModel viewManagerModel;
+  SwitchViewController switchViewController;
+  JPanel views;
+
+  @Before
+  public void setUp() {
```

Comment:
Just test the regular view... why build a new one? Limits line coverage for tests and doesn't seem necessary